pages/api/auth.ts

```diff
@@ -18,21 +20,10 @@ export default async function auth(req: NextApiRequest, res: NextApiResponse) {
          * installed.
          */
 
-        const response = await fetch(
```

Comment:
More dry 👍 . I think we should still keep the fetch in a file within the `pages/api/` directory for the sake of organization. Maybe we could move the `createAppExtension` function out of `lib/appExtensions.ts` and into it's own file `pages/api/auth/createAppExtension.ts` and rename this file to `pages/api/auth/index.ts`

Reply:
It's a great idea 👍🏼 , but if we add it in the `api/auth` folder next.js is going to considere it as an api route, and in case someone access is going to crash. 

Should it be good to place `createAppExtension`  inside `/lib/auth` file ? Noticed  `getAuth` lives there, and we are importing it into our api route --> `api/auth.ts` file 

Otherwise, we could continue keeping individualy the fetch in the corresponding api pages and not using dry, up to us 👍🏼

Reply:
Ah I see what you're saying. My thinking was about keeping all external requests within the api directory, but you're right that we are making requests via node-bigcommerce from lib. I think this is okay.